devito/types/grid.py

```diff
@@ -421,3 +425,63 @@ class Interior(SubDomain):
 
     def define(self, dimensions):
         return {d: ('middle', 1, 1) for d in dimensions}
+
+
+class SubDomains(SubDomain):
```

Comment:
I think we need a different name, as this one is too similar to ``SubDomain``. 

off top of my head: ``SubDomainBundle`` ? Really don't know, we can discuss this offline if you want

Reply:
Yeah, was going to re-consider this. I think `SubDomains` is a good name but (as you say) is too similar.